### ISSUE TITLE
fails on very simple image

### ISSUE THREAD

**Issue by murphyk** [lukas-blecher/LaTeX-OCR#335]
It fails to generate a valid latex string for the simple image below.

Code to reproduce this:
https://colab.research.google.com/gist/murphyk/7727c1ecd1169c16d02600001818c487/latex-ocr-test.ipynb

![280550504-198d0b52-d8ea-4e74-b540-834b3875e15b](https://github.com/lukas-blecher/LaTeX-OCR/assets/4632336/879de825-5acf-4dec-8566-c5ed0c006f49)

**Comment by kevinlinxc** [comment]
I feel like this was meant for actual latex, not handwriting

**Comment by runvnc** [comment]
@murphyk did you find a solution for handwritten math recognition?

**Comment by murphyk** [comment]
no. but google gemini (internal version) is looking promising...
